rpcs3/rpcs3qt/game_list_frame.cpp

```diff
@@ -1492,63 +1575,168 @@ void game_list_frame::ShowContextMenu(const QPoint &pos)
+				disc->setDisabled(true);
+			}
+			else
 			{
-				size_information = tr("Game Directory Size: %0\nCurrent Free Disk Space: %1\n\n").arg(gui::utils::format_byte_size(current_game.size_on_disk)).arg(gui::utils::format_byte_size(stat.avail_free));
+				disc->setChecked(true);
 			}
 		}
+		else
+		{
+			disc->setVisible(false);
+		}
 
-		QMessageBox mb(QMessageBox::Question, tr("Confirm %1 Removal").arg(gameinfo->localized_category), tr("Permanently remove %0 from drive?\n%1Path: %2").arg(name).arg(size_information).arg(qstr(current_game.path)), QMessageBox::Yes | QMessageBox::No, this);
-		mb.setCheckBox(new QCheckBox(tr("Remove caches and custom configs")));
+		caches->setChecked(true);
+		icons->setChecked(true);
+		mb.setCheckBox(disc);
+
+		QGridLayout* grid = qobject_cast<QGridLayout*>(mb.layout());
+		int row, column, rowSpan, columnSpan;
+
+		grid->getItemPosition(grid->indexOf(disc), &row, &column, &rowSpan, &columnSpan);
+		grid->addWidget(caches, row + 3, column, rowSpan, columnSpan);
+		grid->addWidget(icons, row + 4, column, rowSpan, columnSpan);
+		grid->addWidget(save_states, row + 5, column, rowSpan, columnSpan);
+		grid->addWidget(captures, row + 6, column, rowSpan, columnSpan);
+		grid->addWidget(recordings, row + 7, column, rowSpan, columnSpan);
+		grid->addWidget(screenshots, row + 8, column, rowSpan, columnSpan);
 
 		if (mb.exec() == QMessageBox::Yes)
 		{
-			const bool remove_caches = mb.checkBox()->isChecked();
-			if (fs::remove_all(current_game.path))
+			const bool remove_caches = caches->isChecked();
+
+			// Remove data path in "dev_hdd0/game" folder (if any)
+			if (has_data_dir && !RemoveContentPath(data_dir, gameinfo->localized_category.toStdString()))
 			{
-				if (remove_caches)
-				{
-					if (fs::is_dir(cache_base_dir))
-					{
-						if (fs::remove_all(cache_base_dir))
-							game_list_log.notice("Removed cache directory: '%s'", cache_base_dir);
-						else
-							game_list_log.error("Could not remove cache directory: '%s' (%s)", cache_base_dir, fs::g_tls_error);
-					}
+				QMessageBox::critical(this, tr("Failure!"), remove_caches
+					? tr("Failed to remove %0 from drive!\nPath: %1\nCaches and custom configs have been left intact.").arg(name).arg(qstr(data_dir))
+					: tr("Failed to remove %0 from drive!\nPath: %1").arg(name).arg(qstr(data_dir)));
 
-					RemoveCustomConfiguration(current_game.serial);
-					RemoveCustomPadConfiguration(current_game.serial);
-				}
-				m_game_data.erase(std::remove(m_game_data.begin(), m_game_data.end(), gameinfo), m_game_data.end());
-				game_list_log.success("Removed %s %s in %s", gameinfo->localized_category, current_game.name, current_game.path);
-				Refresh(true);
+				return;
 			}
-			else
+
+			// Remove lock file in "dev_hdd0/game/＄locks" folder (if any)
+			RemoveContentBySerial(rpcs3::utils::get_hdd0_dir() + "/game/＄locks/", current_game.serial, "lock");
```

Comment:
```suggestion
			RemoveContentBySerial(rpcs3::utils::get_hdd0_dir() + "game/＄locks/", current_game.serial, "lock");
```

Reply:
In general, I saw in the project (even in this .cpp file) some code using / and some code not using /. I preferred (for more safety) to add the initial /.
Just applying this and all the other proposed changes